connector/src/main/scala/org/apache/spark/sql/riak/DefaultSource.scala

```diff
@@ -81,5 +96,30 @@ object DefaultSource {
     val bucket = parameters(RiakBucketProperty)
     BucketDef(bucket, None)
   }
+  
+  private def parseRiakConnectionOptions(options: Map[String, String], conf: SparkConf): RiakConnector = {
```

Comment:
IMO better to move all 3 helper methods to the corresponding companion objects for RiakConnectorConf,  ReadConf, WriteConf